packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -89,7 +89,7 @@ export function computePerformanceResourceDuration(entry: RumPerformanceResource
 export function computePerformanceResourceDetails(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏The name is not consistent with the other functions. The main concept is "Resource Entry" (ex: computeResourceEntryType, computeResourceEntrySize) but here we talk about "Performance resource". 
Maybe something like 
```ts
export function computeResourceEntryTimings(
  entry: RumPerformanceResourceTiming
): PerformanceEntryTimings | undefined 
```

Reply:
Ah, I missed this!

IIRC `Details` was chosen instead of `Timings` to avoid the confusion with `PerformanceResourceTiming`. I would be enclined to keep `Details` (`computeResourceEntryDetails`). What do you think?